### PR TITLE
Alpha.1 release updates

### PR DESCRIPTION
**Pull request by SteveLasker**
Updates the docs for Alpha.1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,7 +70,8 @@ This project is covered under the Apache 2.0 license. You can read the license [
 
 =======
 
-[notation-releases]:      https://github.com/shizhMSFT/notation/releases/tag/v0.5.0
+[notation-releases]:      https://github.com/notaryproject/notation/releases/tag/v0.7.0-alpha.1
```
Comment (iamsamirzon) [inline README.md:73]: Should we point to this specific notation release, or to the generic release folder https://github.com/notaryproject/notation/releases to keep it compatible with future churns?

**Comment by iamsamirzon** [comment]
A generic comment on using the --plain-http flag with notation. Is that optional or mandatory for alpha.1 ? If mandatory, should it also be added with the below snippet in the Quick Start section of the Readme.MD ?
```
notation sign $IMAGE
notation list $IMAGE 
notation verify $IMAGE
```

**Comment by iamsamirzon** [comment]
LGTM, provided the above comments are addressed

**Comment by SteveLasker** [comment]
Thanks @iamsamirzon, 
`--plain-http` is required if you don't configure insecure registries in the config file. I've captured the issue here: #95 
The feedback for the root readme was to keep it as super simple as possible. The hello-signing doc goes into the detailed steps with descriptions

File: README.md (hunk 1 above)
Comment (SteveLasker) [reply README.md:73]: I had a similar thought. Future churns will have commands that reference new releases. So, I kept the link to the specific release to assure folks don't get broken between releases.

**Review by sajayantony — APPROVED** [review]

**Review by iamsamirzon — APPROVED** [review]

**Review by sajayantony — APPROVED** [review]
Approved updates after rebase.

**Pull request merged**